engine/runtime/src/main/java/org/enso/interpreter/epb/node/ForeignEvalNode.java

```diff
@@ -49,31 +47,30 @@ public static ForeignEvalNode build(
 
   @Specialization
   Object doExecute(
-      VirtualFrame frame,
-      @CachedContext(EpbLanguage.class) ContextReference<EpbContext> contextRef) {
-    ensureParsed(contextRef);
+      VirtualFrame frame) {
```

Comment:
With this change, you can now get rid of `@Specialization`, name this method `execute` and make the node concrete.

Reply:
The sole purpose of this being a specialization was to easily inject the context ref.